### PR TITLE
Server contact bug fix, update-user bug fix, dynamic contacts-forms refactoring, client-server contact flow

### PR DESCRIPTION
**Pull request by mebeborn**

### CODE REVIEW

File: client/src/app/page/my-profile/my-profile.component.scss [hunk 1]
```diff
@@ -41,14 +41,23 @@
...
   .contact_down {
     position: absolute;
-    bottom: 25px;
+    margin-top: 70px;
 
-    :first-child {
-      margin-bottom: 4px;
-    }
+ 
+    // :first-child {
```
Comment (idyac) [inline client/src/app/page/my-profile/my-profile.component.scss]: do we still need this?

File: server/api/v1/controllers/UserController.js [hunk 2]
```diff
@@ -51,14 +51,9 @@ const updateOne = async (req, res) => {
...
-  if (contacts) {
-    contacts = contacts.map(el => ({
-      contact_name: el.contactName,
-      contact_value: el.contactValue,
-    }));
-    user.contacts = [...contacts];
-  }
+  let contacts = req.body.contacts
+
+  user.contacts = [...contacts];
```
Comment (idyac) [inline server/api/v1/controllers/UserController.js]: what about to simplify it? [...req.body.contacts] ?

File: client/src/app/page/create-update-user-page/additional-contacts/contact-types/contact-types.component.scss [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+}
+.dropdown-menu {
+  position: relative;
+  left: 0;
+  right: auto;
+  float: none;
+  margin: -1px;
+  top: 100%;
+  bottom: auto;
+  background-color: #fff;
```
Comment (idyac) [inline client/src/app/page/create-update-user-page/additional-contacts/contact-types/contact-types.component.scss]: move it also to const

**Comment by mebeborn** [comment]
Resolved =)

**Pull request merged**